Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:focal-20230126
+FROM ubuntu:latest
```

Comment:
Instead of relying on a moving tag like `latest` let's use a stable one like `jammy-20230816` so that we get reproducible builds.

Reply:
> It's still not completely reproducible as we are running apt update -y and pip3 install, installing these without explicit versioning on them.

Yes, correct. Nevertheless it's one step closer to reproducible builds and makes the build process a bit easier to reason about.

> Also I think it's worthwhile adding
`RUN pipenv run python3 test_httpbin.py`
after the pipenv sync to run the unit tests during the build. Thoughts?

That would be a reasonable thing to do but IMHO not in a container image which is meant to be run in production.

Ideally we'd add a github workflow job which runs those.